server/athenian/api/align/queries/metrics.py

```diff
@@ -280,64 +316,55 @@ async def calculate_team_metrics(
         pr_metrics, release_metrics, jira_metrics = _triage_metrics(request.metrics)
 
         if pr_metrics:
+            teams_filters = []
+            for (team_id, td), members in zip(request.teams.items(), team_members):
+                repos = _RepositoriesWrapper.from_requested_team_details(td, all_repos)
+                teams_filters.append(
+                    TeamSpecificFilters(
+                        team_id=team_id,
+                        participants={PRParticipationKind.AUTHOR: members},
+                        repositories=repos,
```

Comment:
```suggestion
                        repositories=_RepositoriesWrapper.from_requested_team_details(td, all_repos),
```
(repos is a very common variable, don't want to pollute the locals)